src/contexts/v2/projectContext.ts

```diff
@@ -46,6 +47,9 @@ export type V2ProjectContextType = {
   primaryTerminalCurrentOverflow: BigNumber | undefined
   totalTokenSupply: BigNumber | undefined
 
+  nftRewardsCid: string | undefined
+  nftRewardTiers: NFTRewardTier[] | undefined
```

Comment:
**suggestion**: maybe we can namespace all nft reward properties under a single object:

```suggestion
  nftRewards: {
     cid: string | undefined
     rewardTiers: NFTRewardTier[] | undefined
  }
```

Reply:
Yeah good idea, done